momentum/3/3-reference/3-reference-conf-ref-prefer_ipv6_mxs.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "prefer_ipv6_mxs"
+description: "prefer IPv6 over IPv4 when they have the same priority"
+---
+
+<a name="conf.ref.prefer_ipv6_mxs"></a>
+## Name
+
+prefer_ipv6_mxs — prefer IPv6 addresses over IPv4 addresses when they have the same priority
```

Comment:
I would reword "when they have the same priority" to "at a given MX priority level" here and everywhere else that phrase is used.  It clarifies this is MX priority.